api/src/routes/api.ts

```diff
@@ -65,8 +66,20 @@ router.get("/:category/:job/:image/:size", async function(req, res, next) {
     let size: string = req.params.size;
     let job = getJobFromRequest(req);
     let deriv = await job.getImage(image).derivative(legalSizes[size] ?? "THUMBNAIL");
-    console.log(deriv);
     res.sendFile(deriv);
 });
 
+router.delete("/:category/:job/:image/*"), async function(req, res, next) {
+    //return ImageFile.delete();
+    let image: string = req.params.image;
+    let job = getJobFromRequest(req);
+    let imageObj = job.getImage(image);
+    if (imageObj !== null) {
+        imageObj.delete();
+        res.send(JSON.stringify( { status: 'ok' } ));
+    } else {
+        res.send(JSON.stringify( { status: 'image missing' } ));
```

Comment:
This should also return a 404 status; not sure how to do that, but it's probably not hard.

Reply:
Added 404